pennylane/qnn/torch.py

```diff
@@ -395,3 +404,13 @@ def __str__(self):
     def input_arg(self):
         """Name of the argument to be used as the input to the Torch layer. Set to ``"inputs"``."""
         return self._input_arg
+
+    @staticmethod
+    def set_input_argument(input_name: Text = "inputs") -> None:
+        """
+        Set the name of the input argument.
+
+        Args:
+            input_name (str): Name of the input argument
+        """
+        TorchLayer._input_arg = input_name
```

Comment:
Why is this needed? Is it related to broadcasting?

Reply:
Same question, where is it called?

Reply:
It's not related to broadcasting. The same static method is present in `KerasLayer`, which allows the user to set the name of the input argument to the QNode that needs to be broadcasted. It defaults to `"inputs"`.

Reply:
Cool :)